model-optimizer/extensions/middle/SplitConcatPairToInterpolate.py

```diff
@@ -91,7 +94,11 @@ def replace_interpolate_pattern(graph: Graph, match: dict):
                                        'new_axis_mask': int64_array([0]),
                                        'shrink_axis_mask': int64_array([0]),
                                        'ellipsis_mask': int64_array([0]),
-                                       }).create_node([shape_node, slice_begin, slice_end])
+                                       }).create_node()
```

Comment:
Let's use 'create_op_with_const_inputs' to create the StridedSlice node.

Reply:
Fixed.